.github/workflows/ci.yaml

```
@@ -176,3 +175,15 @@ jobs:
         with:
           name: ecosystem-result
           path: ecosystem-result
+
+  cargo-udeps:
+    name: "cargo udeps"
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: "Install Rust toolchain"
+        run: rustup toolchain install nightly
+      - name: "Install cargo-udeps"
+        uses: taiki-e/install-action@cargo-udeps
+      - name: "Run cargo-udeps"
+        run: cargo +nightly udeps
```

Nit: Do you think it would be useful to output the result to [the job summary](https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#adding-a-job-summary) like this

```suggestion
        run: cargo +nightly udeps | tee $GITHUB_STEP_SUMMARY
```